code/modules/antagonists/traitor/equipment/contractor.dm

```diff
@@ -146,7 +146,7 @@
 /datum/contractor_item/fulton_extraction_kit
 	name = "Fulton Extraction Kit"
 	desc = "For getting your target across the station to those difficult dropoffs. Place the beacon somewhere secure, and link the pack. Activating the pack on your target in space will send them over to the beacon - make sure they're not just going to run away though!"
-	item = /obj/item/storage/box/contractor/fulton_extraction
+	item = /obj/item/storage/box/contractor_fulton_extraction
```

Comment:
why are you changing this?

Reply:
`obj/item/storage/box/contractor` is spawnable and it's an empty box. It's a bit pointless to have `/contractor/` when there's no subtypes under it other than the Fulton extraction.

Also this reduces it down to two types of contractor boxes - the standard loadout and contract kit. I had an issue a while back when I TC traded for a contractor box and the admin had difficulty spawning me the correct box since the type names are unclear and there's several of them.

Reply:
Thinking about this again now I could move them all under this subtype if that's preferred.

Reply:
yea putting it all under 1 subtype is better than having none of them under any subtype when they should be

Reply:
yea putting it all under 1 subtype is better.
There is no reason to not have it under a subtype if its for the contractor, makes it easier to locate and is more organised

Reply:
Moved it all under 1 subtype and updated CL.